.env-template

```diff
@@ -0,0 +1,6 @@
+DEBUG=on
+SECRET_KEY=your-secret-key
+DATABASE_URL=psql://user:un-githubbedpassword@127.0.0.1:8458/database
```

Comment:
Ideally I'd want to copy paste this template to create dev env out of the box, here you have variables that are connected, but have different values from each other (db username, password)

Reply:
I'll corect it in next commit.